examples/fairmotion_interface.py

```diff
@@ -149,38 +153,100 @@ def metadata_parser(self, metadata_dict: Dict[str, Any], to_file: bool):
                     data[k] = mn.Vector3(v)
         return data
 
-    def save_metadata(self, name: str):
+    def save_metadata(self, file: str):
         """
         Saves the current metadata to a json file in given file path
         """
-        filepath = f"{self.metadata_dir}{name + FILE_SUFFIX}.json"
+        # set key for accessing metadata from dict
+        if "user" in self.metadata:
+            name = "user"
+        else:
+            name = "default"
+
+        if file:
+            # building filepath for saving
+            if file == "default":
+                file = METADATA_DIR + "default.json"
+                # print() name = "user"
+
+            elif os.path.exists(file):
```

Comment:
This can be better handled by globbing, right? Or iterating through a list of possible filenames. The logic is a bit complex as of now.